modules/gltf/gltf_document.cpp

```diff
@@ -6425,61 +6431,189 @@ GLTFAnimation::Track GLTFDocument::_convert_animation_track(Ref<GLTFState> p_sta
 	Vector<real_t> times;
 	times.resize(key_count);
 	String path = p_animation->track_get_path(p_track_i);
+	if (path.begins_with("%")) {
+		const Vector<String> node_suffix = String(path).split(":");
+		String path_split = node_suffix[0];
+		Node *node = p_animation_player->get_owner()->get_node_or_null(path_split);
+		path = String(p_animation_player->get_parent()->get_path_to(node)) + ":" + node_suffix[1];
+	}
```

Comment:
same here. This special case should not be needed, and hence the extra "p_animation_player" argument should also not be necessary.

Reply:
Modified to be done this way.

```
	String path = p_animation->track_get_path(p_track_i);
	const Vector<String> node_suffix = String(path).split(":");
	String path_split = node_suffix[0];
	Node *node = p_animation_player->get_owner()->get_node_or_null(path_split);
	// Rewites the path to remove unique names.
	path = String(p_animation_player->get_parent()->get_path_to(node)) + ":" + node_suffix[1];
```

Reply:
Removing rewrite path.